arrocke/gloss-translation
# refactor: active styles for nav links

## What has changed

* Added background to nav header
* Added active styles to nav links
* Moved translation view from `/languages/{code}` to `/interlinear/{code}` - this allows the primary nav to distinguish between the manage language view and the translation view for determining the active link
* Renamed app in header to "Global Bible Tools"



## Connected Issues

closes #283 



## QA Steps



1. Open app - notice that interlinear link is highlighted
2. Navigate to language or user list - notice that highlighted link changes
